### PR TITLE
A round of Kobo specific script cleanup

### PR DESCRIPTION
**Pull request by NiLuJe**
Mostly revolving about updating/slimming down the startup & nickel script now that I finally fixed KFMon ;p.

See individual commits for more details.

Tested on an H2O, FW 4.7, latest KO beta at the time (v2015.11-1644).

### CODE REVIEW

**Comment by NiLuJe** [comment]
Oh, yeah, relevant user-facing bit: updated the install instructions to point to modern launchers (KSM09, @Baskerville's [fmon](https://github.com/baskerville/fmon), [KFMon](https://github.com/NiLuJe/kfmon), and to strongly discourage people from using the legacy fmon implementation.

Bonus: we get to drop the KoboRoot tarball!

File: platform/kobo/nickel.sh [hunk 1]
```diff
@@ -16,7 +16,7 @@ export LD_LIBRARY_PATH="/usr/local/Kobo"
 # NOTE: That check is possibly wrong on PLATFORM == freescale (because I don't know if the sdio_wifi_pwr module exists there), but we don't terribly care about that.
 if lsmod | grep -q sdio_wifi_pwr; then
     killall udhcpc default.script wpa_supplicant 2>/dev/null
-    [ "${WIFI_MODULE}" != 8189fs ] && wlarm_le -i ${INTERFACE} down
+    [ "${WIFI_MODULE}" != "8189fs" ] && wlarm_le -i ${INTERFACE} down
```
Comment (Frenzie) [inline platform/kobo/nickel.sh]: You really like quotation marks, haha.

File: platform/kobo/fmon/README.txt [hunk 2]
```diff
@@ -1,19 +1,23 @@
 Installation instructions for Kobo:
 
 
-Newer installation based on Start Menu:
-- Install "Start Menu" (http://www.mobileread.com/forums/showthread.php?t=255978)
-- Extract the koreader folder of the zip into the ".adds" directory (both KoboRoot.tgz and the koreader.png file are only needed for the older installation based on Filemonitor. See below.).
+Main installation method based on Start Menu:
```
Comment (Frenzie) [inline platform/kobo/fmon/README.txt]: Oh, I normally never look in the fmon folder. I have been updating the install instructions here: https://github.com/koreader/koreader/wiki/Installation-on-Kobo-devices

(I prefer the "*Recommended* installation method based on Start Menu" phrasing)

File: platform/kobo/fmon/README.txt [hunk 3]
```diff
@@ -1,19 +1,23 @@
...
+Alternate installation method based on fmon:
+- Put the image included in the zip (called "koreader.png") in an "icons" folder created in the main folder of your Kobo and disconnect it from your computer. Add the icon to a collection, set the view to List View and open the icon, then restart your device.
+- Install "fmon" (https://github.com/baskerville/fmon).
+- Extract the remaining content of the zip into the ".adds" directory (the koreader folder).
 
-Selecting the KOReader icon in your home, you will be able to launch KOReader. Simple enough, isn't it? Just be sure to process the image properly (follow those steps carefully) if it's the first time you install koreader, if you made a factory reset or if you deleted it in some way, otherwise you could end up launching koreader automatically at every boot, being unable to exit it without a factory reset.
+Selecting the KOReader icon in your home, you will be able to launch KOReader. Simple enough, isn't it? Just be sure to process the image properly (follow those steps carefully) if it's the first time you install koreader, after a factory reset or if you deleted it in some way, in order to make sure KOReader starts properly.
```
Comment (Frenzie) [inline platform/kobo/fmon/README.txt]: Seems a bit wordy in the first sentence. :-P

Quick suggestion:

Select the KOReader icon in your home to launch KOReader. Simple enough, isn't it? Just be sure to process the image properly. Follow these steps carefully if it's your first time installing KOReader, after a factory reset, or if you deleted it in some way. This will ensure KOReader starts properly.

File: platform/kobo/fmon/README.txt [hunk 4]
```diff
@@ -1,19 +1,23 @@
...
 
-When you update koreader, it should be sufficient to extract the koreader folder, without KoboRoot.tgz - that one is needed to add the launcher.
+When you update koreader manually, it should be sufficient to simply extract the koreader folder.
```
Comment (Frenzie) [inline platform/kobo/fmon/README.txt]: KOReader (at least the first one)

File: platform/kobo/nickel.sh [hunk 5]
```diff
@@ -16,8 +16,8 @@ export LD_LIBRARY_PATH="/usr/local/Kobo"
 # NOTE: That check is possibly wrong on PLATFORM == freescale (because I don't know if the sdio_wifi_pwr module exists there), but we don't terribly care about that.
 if lsmod | grep -q sdio_wifi_pwr; then
     killall udhcpc default.script wpa_supplicant 2>/dev/null
-    [ "${WIFI_MODULE}" != "8189fs" ] && wlarm_le -i ${INTERFACE} down
-    ifconfig ${INTERFACE} down
+    [ "${WIFI_MODULE}" != "8189fs" ] && wlarm_le -i "${INTERFACE}" down
```
Comment (Frenzie) [inline platform/kobo/nickel.sh]: Oh sure, quote the non-var but forget the vars. ShellCheck was right! ;-D

File: platform/kobo/fmon/README.txt (hunk 2 above)
Comment (NiLuJe) [reply platform/kobo/fmon/README.txt]: Yeah, given how outdated that file looked, I'm grateful there's something maintained elsewhere ;D.

File: platform/kobo/fmon/README.txt (hunk 3 above)
Comment (NiLuJe) [reply platform/kobo/fmon/README.txt]: Yeah, that's my curse ^^. Thanks for the suggestion, will use it.
